src/cmd/run.go

```diff
@@ -529,6 +529,9 @@ func callFlatpakSessionHelper(container podman.Container) error {
 		return nil
 	}
 
+	fmt.Fprintf(os.Stderr, "Warning: container %s uses deprecated features\n", container.Name())
+	fmt.Fprintf(os.Stderr, "Consider recreating it with Toolbox version 0.0.97 or newer.\n")
```

Comment:
We should also update the recommended version a few lines above from 0.0.17 to 0.0.97, because containers created with 0.0.17 to 0.0.96 are now deprecated.